site2/docs/reference-metrics.md

```diff
@@ -515,6 +516,27 @@ All the schema metrics are labelled with the following labels:
 | pulsar_schema_get_ops_latency | Summary | Latency of successful operations to get schemas. |
 | pulsar_schema_put_ops_latency | Summary | Latency of successful operations to send schemas. |
 
+### Offload metrics
+
+All the offload metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you configured in `broker.conf`.
+- *namespace*: `namespace=${pulsar_namespace}`. `${pulsar_namespace}` is the namespace name.
+- *topic*: `topic=${pulsar_topic}`. `${pulsar_topic}` is the topic name.
+
+| Name                                           | Type    | Description                                                                     |
+|------------------------------------------------|---------|---------------------------------------------------------------------------------|
+| brk_ledgeroffloader_offload_error              | Counter | The number of failed operations to offload.                                     |
+| brk_ledgeroffloader_offload_rate               | Gauge   | The rate of offloading(bytes/s).                                                |
```

Comment:
```suggestion
| brk_ledgeroffloader_offload_rate               | Gauge   | The rate of offloading (bytes/second).                                                |
```

Reply:
@Anonymitaet Makes sense. I will update accordingly.
Can you pls add it to the Pulsar writing style guide? There are many similar occurrences like "bytes/s".

Reply:
Sure, added here: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.esa4s02v3adl